packages/cdktf-cli/bin/cmds/ui/terraform-context.tsx

```diff
@@ -184,6 +186,52 @@ export const parseOutput = (str: string): DeployingResource[] => {
   }, new Array());
 };
 
+const isObjectEmpty = (obj: Record<string, any>): boolean => {
+  if (typeof obj !== "object") {
+    return false;
+  }
+  return (
+    Object.keys(obj).length === 0 ||
+    Object.values(obj).every(
+      (v) => v === undefined || v === null || isObjectEmpty(v)
+    )
+  );
+};
+
+export const getConstructIdsForOutput = (
+  stackContent: Record<string, any>,
+  output: { [key: string]: TerraformOutput }
```

Comment:
Commenting here but it affects more parts of the PR:
I'd advocate for using `outputs` as name if we handle multiple outputs and `output` only if we are talking about a single one. Currently we seem to use `output` for single and multiple ones which confuses me when reading the code. Luckily we have types that make this clear, but I'd still like the naming to reflect this, too.
As the Terraform command is called `output` I'd keep the name of our `Output` (root) react component the way it is but would rename the other one to `Outputs` as it renders multiple outputs.
I'm not having a particularly strong opinion about this, so if there are good reasons to keep it as is, I'm fine with that, too.
P.S. The Terraform codebase also calls the map [outputs](https://github.com/hashicorp/terraform/blob/main/internal/command/views/json/output.go#L21) 🤓😁